packages/main/src/TableCellBase.ts

```diff
@@ -49,6 +49,10 @@ abstract class TableCellBase extends UI5Element {
 	getFocusDomRef() {
 		return this;
 	}
+
+	isTableCellBase() {
```

Comment:
isTableCell

Reply:
@nnaydenow linked me this PR: https://github.com/SAP/ui5-webcomponents/pull/8822 referencing point 4.

Seems like the main issue is scoping, when you have multiple different instances of the framework in your application, instanceof is not that reliable anymore?